requirements.txt

```diff
@@ -1,3 +1,10 @@
 pyinstaller==6.12.0
 typing-extensions==4.12.2
 viam-sdk==0.41.0
+opencv-python
+pillow
+
+pytest
+pytest-asyncio
+pylint
```

Comment:
Two requests. 
1. Pin a specific version of each of these. Otherwise, a year from now some new employee is going to install these requirements and the code will work on your machine and fail on the new employee's machine because these dependencies had a breaking change between now and then. 
2. Alphabetize this file, so it's easier to check if a certain dependency is already in the list.

Both of these can be done by running `pip freeze` and putting its output in this file. (that approach will also include all the indirect dependencies, which I don't really care about one way or the other)